tests/Concerns/WithMappedCellsTest.php

```diff
@@ -38,8 +38,14 @@ public function can_import_with_references_to_cells()
             public function mapping(): array
             {
                 return [
-                    'name'  => 'B1',
```

Comment:
Can you add new test cases instead of updating the existing one, now I can't see if the original use-case still works like expected without breaking changes

Reply:
Done!